File: check_flags.js
const Database = require('better-sqlite3');
const path = require('path');

// Uygulama adı "s" olduğu için accounts.db yolu bu
const dbPath = path.join('C:\\Users\\sefao\\AppData\\Roaming\\s\\accounts.db');
console.log('DB Path:', dbPath);

try {
    const db = new Database(dbPath, { readonly: true });
    const rows = db.prepare('SELECT id, site, username, bonus_flagged FROM accounts ORDER BY site, id').all();

    console.log('\n=== TUM HESAPLAR ===');
    rows.forEach(r => {
        const flagStatus = r.bonus_flagged ? `FLAG VAR - ${new Date(r.bonus_flagged).toLocaleString('tr-TR')}` : 'Temiz';
        console.log(`ID:${r.id} | ${(r.site + '          ').substring(0, 12)} | ${(r.username + '                    ').substring(0, 20)} | ${flagStatus}`);
    });

    const matbet = rows.filter(r => r.site === 'matbet');
    const matbetFlagged = matbet.filter(r => r.bonus_flagged > 0);

    console.log('\n=== OZET ===');
    console.log(`Toplam hesap: ${rows.length}`);
    console.log(`Matbet hesap: ${matbet.length}`);
    console.log(`Flagli Matbet: ${matbetFlagged.length}`);
    if (matbetFlagged.length > 0) {
        console.log('\nFlagli olanlar:');
        matbetFlagged.forEach(r => console.log(`  -> ${r.username} (ID:${r.id})`));
    }

    db.close();
} catch (e) {
    console.error('HATA:', e.message);
}
process.exit(0);

File: check_flags.py
import sqlite3
import datetime

conn = sqlite3.connect(r'C:\Users\sefao\AppData\Roaming\s\accounts.db')
rows = conn.execute("SELECT id, site, username, bonus_flagged FROM accounts ORDER BY site, id").fetchall()
conn.close()

flagged = [r for r in rows if r[3]]
print("FLAGLI HESAPLAR:")
for r in flagged:
    ts = datetime.datetime.fromtimestamp(r[3]/1000).strftime('%d.%m %H:%M')
    print(r[0], r[1], r[2], ts)

matbet_all = [r for r in rows if r[1] == 'matbet']
print("\nTUM MATBET:")
for r in matbet_all:
    ts = datetime.datetime.fromtimestamp(r[3]/1000).strftime('%d.%m %H:%M') if r[3] else 'Temiz'
    print(r[0], r[2], ts)
